docs/source/getting_started.rst

```diff
@@ -51,16 +51,18 @@ To compute fluid dynamic quantities and simulation metrics, you may execute the
     $ python automatedPostProcessing/compute_flow_and_simulation_metrics.py --case simulation/results_artery/artery/data/[RUN_NUMBER]/Solutions
 
 As default, ``compute_flow_and_simulation_metrics.py`` computes the average values over full cycle, for a desired number of cycles, determined by the ``--start-cycle`` flag. Setting ``--start-cycle 2`` would correspond to computing the averaged values from the second cardiac cycle and onward.
-Alternatively, the user may supply the specific times :math:`t` during the cardiac cycle :math:`T`, such that :math:`t \in [0,T]`, to the ``--times-to-average`` flag. Thus, to compute the metrics at :math:`t=0.951`, and skip the first cycle, the user may run the following command::
+Alternatively, the user may supply the specific times :math:`t` during the cardiac cycle :math:`T` to the ``--times-to-average`` flag, to compute phase averaged quantities, where :math:`t \in [0,T)`. Thus, to compute the metrics at :math:`t=0.2` s representing peak systole, and to skip the first cycle, the user may run the following command::
 
     $ python automatedPostProcessing/compute_flow_and_simulation_metrics.py --case simulation/results_artery/artery/data/[RUN_NUMBER]/Solutions --start-cycle 2 --times-to-average 951
```

Comment:
By the above command and using default values: 
`time_step_to_average = int(t / dt / save_frequency)` in Line 48 will be 2 s not 0.2 s, right? 
In that respect, what would be the outcome of `time_steps_to_average = [time_step_to_average + saved_time_steps_per_cycle * i for i in range(N_tmp)][ start_cycle - 1:]`  ?

Reply:
Thanks @hkjeldsberg!
By assuming as you said, cardiac_cycle=1000 ms, dt=0.1 s, save_freq=5, we have 2000 saved sampled data at each cycle, right? so if one wants to get average only for 2nd cycle by t=0.2 s and if set time_steps_to_average = [2400], it will average over the saved sampled data with t=0.2 s frequency? I may confuse my self. I was wondering to get average at each cycle for example with different number of samples.

Can we calculate averaged velocity at each cycle beforehand ?

Reply:
Hi @ehsannkh. 
Partly correct. If you set time_steps_to_average = [2400] you will only get the instantaneous value at t=1.2s, not averaged. I'm not sure if you can get an average at each cycle (?). 

The averaged velocity is currently computed in the variable `u_avg = Function(V)`, which is the average velocity, averaged over the cycles. The average velocity computed in earlier versions was time averaged  i.e. it computed u_mean(x), where as the newly implemented `u_avg` is a function of both space and time; u_avg(x,t).

Reply:
Great! Thanks @hkjeldsberg!
So one has to give the all the time instances in time_steps_to_average to get average over? for example, `time_steps_to_average = [2400, 4400, 6400, 8400]` gives the average between the results  of 1.2, 2.2, 3.2 and 4.2 ?

Reply:
@ehsannkh Exactly! But right now the script should compute `time_steps_to_average` automatically based on the value you supply, e.g. `--times to average 200` to average over t=0.2. It will then average the value at t=1.2, t=2.2, t=3.2 and t=4.2, as like you mentioned, given that you start at the second cardiac cycle for averaging (`--start-cycle 2`)

Reply:
Great! Thanks! @hkjeldsberg